extension/background/MusicService.js

```diff
@@ -0,0 +1,69 @@
+import * as serviceList from "../../background/serviceList.js";
```

Comment:
This should be using the existing file in `extension/background/musicService.js` (if you don't see that file you'll need to merge master into your branch/fork)

Reply:
Hi @ianb . Thank you for your corrections. Would work on it now.